### PR TITLE
Include Gaia store

### PR DESCRIPTION
**Pull request by AcidLeroy**
LevelDB for Gaiastorage.

- https://github.com/blockstack/gaia

### CODE REVIEW

**Comment by AcidLeroy** [comment]
I just realized there are instructions for doing this. I'll pull down the project to update the README.

**Comment by AcidLeroy** [comment]
K, should be good to go now. Sorry about that!

File: modules/stores.json [hunk 1]
```diff
@@ -113,6 +113,10 @@
     "s3leveldown": {
       "github": "loune/s3leveldown",
       "type": "[AWS S3](https://aws.amazon.com/s3/)"
+    }, 
+     "gaiadown-ts": {
+      "github": "acidleroy/gaidown-ts",
```
Comment (vweevers) [inline modules/stores.json]: Typo

```suggestion
      "github": "acidleroy/gaiadown-ts",
```

**Comment by AcidLeroy** [comment]
I lied, I wasn't right. I can't type today apparently. I'm trying again, and hopefully this time there won't be any issues.

**Pull request merged**